interface-definitions/include/interface/duid.xml.i

```diff
@@ -0,0 +1,15 @@
+<!-- include start from interface/duid.xml.i -->
+<leafNode name="duid">
+  <properties>
+    <help>DHCP unique identifier (DUID) to be sent by DHCP client</help>
+    <valueHelp>
+      <format>duid</format>
+      <description>DHCP unique identifier (DUID)</description>
+    </valueHelp>
+    <constraint>
+      <validator name="duid"/>
```

Comment:
Why create a discrete file and not only write:

`<regex>([0-9A-Fa-f]{2}:){,127}([0-9A-Fa-f]{2})</regex>`?

Reply:
This was based on the feedback here: https://github.com/vyos/vyos-1x/pull/2650#discussion_r1430530826. I read your intention on that one as having common `<leafNode name="duid"/>` which can be reused in DHCP client and server config alike. Did I misinterpret the intent?

Reply:
Update: Re-reading your message, do you mean we can remove the separate `validator/duid` file and just inline the `regex`?